zipline-gradle-plugin/src/main/kotlin/app/cash/zipline/gradle/ZiplinePlugin.kt

```diff
@@ -73,10 +70,11 @@ class ZiplinePlugin : KotlinCompilerPluginSupportPlugin {
       createdTask.outputDir.set(linkOutputDir.parentFile.resolve("${linkOutputDir.name}Zipline"))
     }
 
-    project.tasks.withType(KotlinWebpack::class.java).configureEach { kotlinWebpack ->
-      if (kotlinBinary.mode.toString().equals(kotlinWebpack.mode.toString(), ignoreCase = true)) {
-        kotlinWebpack.dependsOn(compileZiplineTaskName)
-      }
+    val serveTaskName = "serve${kotlinBinary.mode.toString().lowercase().capitalized()}Zipline"
```

Comment:
`.lowercase().capitalized()` isn't the lowercase redundant?

Reply:
Unfortunately not. The enum returns an uppercased string and captialized only ensures the first character is uppercase and doesn't care whether the rest of the string is upper or lower case.